lib/matplotlib/widgets.py

```diff
@@ -678,6 +679,11 @@ class TextBox(AxesWidget):
         The color of the text box when hovering.
     """
 
+    @cbook.deprecated("3.3")
+    @property
+    def params_to_disable(self):
+        return [key for key in mpl.rcParams if 'keymap' in key]
```

Comment:
Is this deprecation part of this?

Reply:
yes, we're not using that attribute anymore

Reply:
I guess it should ever have been public?

Reply:
yup, the usual thing.